tests/wpt/meta/MANIFEST.json

```diff
@@ -365341,7 +365375,9 @@
          "=="
         ]
        ],
-       {}
+       {
+        "testdriver": true
+       }
```

Comment:
Do you know what's going on with the manifest updates here?

Reply:
I ran `./mach update-manifest` while there was a merge conflict in it. I think the tool then fully updates the file, rather than only the changed parts. I think the same would happen on `main`. They seem to be fine to keep in there, but I don't know the implication of these arguments.